Quote_Manager_CLI_Peter/src/cli.py

```diff
@@ -0,0 +1,227 @@
+import click
+import json
+import random
+import logging 
+from sqlalchemy.orm import Session
+from src.models import Quote
+from src.database import get_db
+from src.utils import query_existing_data, load_data
+import os
+
+def create_log_file(log_file_name:str, error_log_file_name:str, var_dir:str):
+    """
+    Function to create a log file
+
+    """
+    if isinstance(log_file_name, str) == True \
+        and isinstance(error_log_file_name, str) == True \
+            and isinstance(var_dir, str) == True:
+        
+        if log_file_name.endswith(".log") == True \
+            and error_log_file_name.endswith(".log") == True:
+
+            curr_dir = os.path.dirname(os.path.abspath(__file__))
+            base_dir = os.path.dirname(curr_dir)
+            var_dir = os.path.join(base_dir, var_dir)
+
+            if not os.path.exists(var_dir):
+                os.makedirs(var_dir)
+            log_dir = os.path.join(var_dir, "log")
+            if not os.path.exists(log_dir):
+                os.makedirs(log_dir)
+            
+            log_file_path = os.path.join(log_dir, log_file_name)
+            if not os.path.exists(log_file_path):
+                with open(log_file_path, "w") as f:
+                    f.write("")
+            
+            error_log_file_path = os.path.join(log_dir, error_log_file_name)
+            if not os.path.exists(error_log_file_path):
+                with open(error_log_file_path, "w") as f:
+                    f.write("")
+
+            return log_file_path, error_log_file_path
+        else:
+            raise ValueError("Invalid file format. Only log files are allowed")
+
+    else:
+        raise ValueError("Invalid input arguments. Input arguments must be strings")
+
+
+log_file_name = "quote_manager.log"
```

Comment:
It's a good convention to setup logs at the top of the file after imports or before actual scripts because you might need to log any part of that script